README.md

```diff
@@ -25,9 +25,16 @@ conda activate ortho_seq
 ## Then, install the package:
 ``python setup.py install``
 
-## Gather the input files needed.
-1. You'll have one file with sequence data as seen in the first column in the image above (.txt or .dat or .csv). See repo's data folder for examples of what these look like for DNA or proteins.
-2. You'll have one file with corresponding phenotypes as seen in the last column in the image above (.txt or .dat or .csv) with the same length as the number of sequences. Phenotypes here are defined as real numbers (see paper linked above for more background on this).
+## Gather the input file(s) needed.
+
+There are three main ways to submit your sequence and phenotype files to *ortho_seqs*. The first method is to submit them separately, in their own .txt files. Recently, however, an update was added that allows you to submit them both in the same file. For this to apply:
```

Comment:
Looks like GUI doesn't support this yet a combination of file, perhaps make a note on this